src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+
+FID is very fast for the average CMS website on desktop – only Wix scores lower than 100% – and mixed on mobile. Most CMSs deliver mobile FID on an average site within a reasonable range of the desktop score; for Wix the number of websites that have a good FID on mobile is nearly half the desktop total.
+
+{{ figure_markup(
+  image="cms-real-user-first-input-delay-experiences.png",
+  caption="Real-user First Input Delay experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" First Input Delay experience. All have a 100% experience score on desktop, except Wix with 87%. For mobile Wordpress has 88%, Drupal 76%, Joomla 71%, Squarespace 91% and Wix 46%.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=893606466&format=interactive",
+  sheets_gid="465267881",
+  sql_file="core_web_vitals_distribution.sql"
+  )
+}}
+
+The FID scores are generally good here, but this does not translate to similar LCP scores. As suggested, the weight of individual pages on CMSs in addition to mobile connection quality, could play a role in the performance gaps that we see here. There is a small margin of difference between the resources shipped to desktop and mobile versions of a website. Last year we noted that optimizing for the mobile experience was necessary. Average scores have increased on desktop and mobile, but further attention is required here.
+
+**Cumulative Layout Shift**
+
+Cumulative Layout Shift (CLS) measures the instability of content on a web page within the first 500ms of user input. CLS measures any layout changes which happen after user input. This is important on mobile in particular, where the user will tap where they want to take an action – such as a search bar – only for the location to move as additional images, ads, or similar load.
+
+A score of 0.1 or below is good, over 0.25 is poor, and anything in between needs improvement.
+
+{{ figure_markup(
+  image="cms-real-user-cummulative-layout-shift-experiences.png",
+  caption="Real-user Cummulative Layout Shift experiences.",
+  description='Bar chart showing the top 5 CMSs and whether they have a "good" First Input Delay experience. Wordpress has a 47% of desktop sites with a "good experience" and 57% of mobile sites. Drupal has 58% for desktop and 70% for mobile, Joomla has 51% for desktop and 63% for mobile, Squarespace has 35% for desktop and 44% for mobile, and Wix has 58% for desktop and 59% for mobile.',
```

Comment:
Update for CLS

Reply:
Fixed. Stats were good but missed the update after "good".